docs/application/native/guides/app-management/app-controls.md

```diff
@@ -109,8 +109,9 @@ app_control_destroy(app_control);
 ```
 > **Note**
 >
-> From Tizen 5.0 and onwards, two APIs whose callbacks have a result were added to return the result.
-> First among the two APIs is `app_control_send_launch_request_async()` that asynchronously launches application controls, while other API is `app_control_send_launch_request_sync()` that synchronously launches application controls.
+> Two APIs that can obtain a reply result were added to return the result.
```

Comment:
@hjhun  How about this? 

Since we get a callback response after a callback 
The following APIs that obtain a callback response were added to return the result:

Reply:
The app_control_send_launch_request_sync() doesn't have a callback function. Please check the API signature.

Reply:
@hjhun 
In context to your original text i.e 
"two APIs whose callbacks have a result were added to return the result." 
I added callback, 
Fine changed as per your latest comment. 
Now it seems fine